app.py

```diff
@@ -393,7 +392,12 @@ def handle_settings():
     user = get_database_entry(session["username"])
 
     if request.method == "GET":
-        return json.dumps(dict(user.items()))
+        userPrivacyDictRaw = dict(user.items())
```

Comment:
Per Google style guide, the format for Python variables is "snake case", e.g., `user_privacy_dict_raw`